doc/content/with_conda.md

```diff
@@ -21,16 +21,19 @@ OpenMC separately, or using its conda environment (i.e., do
 
 !alert! note title=tldr
 
-All that you need to compile Cardinal is:
+Once your `cardinal` conda environment is set up, all you need to do to compile Cardinal is:
 
 ```
-conda activate moose
+conda activate cardinal
+conda install -c conda-forge openmc # Install OpenMC from conda-forge
 cd $HOME
 git clone https://github.com/neams-th-coe/cardinal.git
 cd cardinal
 ./scripts/get-dependencies.sh
 export ENABLE_NEK=false
 export HDF5_ROOT=$CONDA_PREFIX
+export OPENMC_DIR=$CONDA_PREFIX
```

Comment:
suggest to move this to a "tip" box, if anyone wants to use OpenMC conda and MOOSE conda simultaneously

`!alert tip`